base_layer/wallet_ffi/Cargo.toml

```diff
@@ -3,7 +3,7 @@ name = "tari_wallet_ffi"
 authors = ["The Tari Development Community"]
 description = "Tari cryptocurrency wallet C FFI bindings"
 license = "BSD-3-Clause"
-version = "0.20.0"
+version = "0.21.0"
```

Comment:
Please leave incrementing the version to Mike. His process for producing a release will deal with this and it gets incremented based on the Conventional Commit tag. In this case if this PR is breaking then there should be a ! in the title.

Reply:
I sometimes overlook this accidentally so have made a point to remember it. Will revert it in light of the above though, thanks.